ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/restore/KsqlRestoreCommandTopic.java

```diff
@@ -278,6 +278,7 @@ public static void main(final String[] args) throws Exception {
         ProducerConfig.ACKS_CONFIG,
         "all"
     );
+    transactionalProperties.putAll(serverConfig.originalsWithPrefix("ksql.server.command.consumer."));
```

Comment:
In KsqlRestConfig, can you make this string var public so you can reference it here?
```
  private static final String COMMAND_CONSUMER_PREFIX =
      KSQL_CONFIG_PREFIX + "server.command.consumer.";
```

Reply:
updated

Reply:
Can you update the string to reference COMMAND_CONSUMER_PREFIX instead?